src/components/ToDo/styles/LastRow.styled.ts

```diff
@@ -0,0 +1,72 @@
+import styled from 'styled-components'
+
+export const Rectangle = styled.div<{ isDarkMode: boolean }>`
+  height: 30px;
+  background-color: ${(props) => (props.isDarkMode ? 'hsl(235, 24%, 19%)' : 'hsl(0, 0%, 98%)')};
+  border-radius: 5px;
+  color: ${(props) => (props.isDarkMode ? 'hsl(234, 39%, 85%)' : 'hsl(235, 19%, 35%)')};
+  box-shadow: ${(props) => (props.isDarkMode ? '2px 2px 4px #000000' : '')};
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 25px 15px 25px 15px;
+`
+export const Circle = styled.button`
```

Comment:
consider naming this something like CircleButton instead. makes it easier to understand what this component is